linux/ibus-keyman/src/keymanutil.h

```diff
@@ -3,9 +3,9 @@
 /*
  * Keyman Input Method for IBUS (The Input Bus)
  *
- * Copyright (C) 2018, 2020 SIL International
+ * Copyright (C) 2018-2023 SIL International
  *
- * kmflutil is dual licensed under the MIT or GPL licenses as described below.
+ * keymanutil is dual licensed under the MIT or GPL licenses as described below.
```

Comment:
If KMFL is removed, does this mean we only  use the MIT license now?

Reply:
No, there are still files under GPL.